enarx-keep-sgx-shim/src/event.rs

```diff
@@ -24,9 +27,13 @@ pub extern "C" fn event(
     // Exception Vector Table
     match h.aex.gpr.exitinfo.exception() {
         Some(Exception::InvalidOpcode) => {
-            match unsafe { core::slice::from_raw_parts(h.aex.gpr.rip as *const u8, 2) } {
+            match unsafe {
+                core::slice::from_raw_parts(h.aex.gpr.rip as *const u8, 2)
+                    .try_into()
+                    .unwrap()
```

Comment:
Small comment here that unwrapping in the `sgx-shim` can cause an [infinite loop](https://github.com/enarx/enarx/blob/master/enarx-keep-sgx-shim/src/main.rs#L13), so we prefer not to unwrap.

Reply:
Thank you both, very good to know!

I'm thinking it would be better to use slices for the opcode constants, like:
`pub const OP_SYSCALL: &[u8] = &[0x0f, 0x05];` 
so we don't risk that loop and also maintain the original flexibility of slice pattern matching.

As for a good crate for the opcode constants, my first instinct is to use `intel-types`, but I don't know if that is necessarily appropriate because of issue #442 . Would it be better to define them local to the `enarx-keep-sgx-shim` crate, similar to the ELF `Header` struct and `MAGIC` const?